src/functions/list_authors.rb

```diff
@@ -0,0 +1,7 @@
+def list_authors
+  puts 'List of Authors:'
+  puts 'No authors in the catalog' if authors.empty?
+  labels.each do |element|
+    puts "ID: #{element.id} - First Name: #{element.first_name} - Last Name: #{element.last_name}"
+  end
+end
```

Comment:
- [] Kindly check when you list authors, the app doesn't show any message even if you create an author:
![image](https://user-images.githubusercontent.com/103154117/219091778-257d67bf-cbc7-4e96-b6c5-1a13e9217e3f.png)

Reply:
Thank you. I will fix it.